docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
 
-### Menu File
+## Shift in Nuke
 
-In the *menu.py*.py file it is required to configure the Shift *menu.py* and toolbar entries in the Nuke UI.
+When the requirements are added you will Shift a Shift menu entry in the toolbar and top menu inside Nuke.
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">import os
-import nuke
+This entry will contain the access to open the Shift UI and to create a ShiftWorkflow Node.
 
-try:
-    import shift
-    shiftPath = os.path.dirname(shift.__file__)  # Get the directory location of Shift
-    # Build the path to the shift_icon for the toolbar
-    iconPath = os.path.join(shiftPath, "icons", "shift_icon_window.png")
-    shiftCode = "from shift.tools import nshift;nshift.show()"
-    # Create the toolbar entry
-    menubar = nuke.*menu.py*("Nodes")
-    shiftM = menubar.addMenu("Shift", icon=iconPath)
-    shiftM.addCommand("Shift UI", shiftCode, '')
-    # Create a *menu.py* entry
-    menubar = nuke.*menu.py*("Nuke")
-    shiftM = menubar.addMenu("Shift")
-    shiftM.addCommand("Shift UI", shiftCode, '')
-except Exception as e:
-    pass  # Skip the Error raise to avoid Nuke failing at opening if setting up the *menu.py* does not work on startup.
-</code></pre>
+<figure>
+      <img src="images/nuke_shift_toolbar.png" alt="Shift Toolbar">
+      <figcaption><b>Figure 1</b>: Shift tools in Nuke nodes menu.</figcaption>
+</figure>
+
+<figure>
+      <img src="images/nuke_shift_menu.png" alt="Shift menu">
+      <figcaption><b>Figure 2</b>: Shift UI on Nuke's top menu bar.</figcaption>
+</figure>
 
-This code, placed in a *menu.py* file and executed by Nuke, will create a Shift toolbar entry (Figure 1) and a Shift *menu.py* entry (Figure 2) when Nuke is initialized.
+you will be able to create ShiftWorkflow nodes usign the tab key in the node graph too.
 
 <figure>
-      <img src="images/nuke_shift_toolbar.png" alt="Shift Toolbar">
-      <figcaption><b>Figure 1</b>: Shift *menu.py* entry in the Nuke Nodes Toolbar.</figcaption>
+      <img src="images/nuke_shift_search_node.png" alt="Creating ShiftWorkflow node.">
+      <figcaption><b>Figure 3</b>: Create ShiftWorkflow nodes in node graph window.</figcaption>
+</figure>
+
+## ShiftWorkflow Node
+
+The Shift plugin for Nuke includes a node to execute Shift workflow's in the Nuke's node graph.
+
+In the Shiftworkflow node you can select you Shift workflow to load it in the node and automatically generate the required input and output knobs and inputs. (If auto plugs option is checked.)
+
+<figure>
+      <img src="images/nuke_shift_node_propierties.png" alt="ShiftWorkflow propierties.">
+      <figcaption><b>Figure 3</b>: Main Propierties tab of ShiftWorkflow node.</figcaption>
 </figure>
 
+You can use relative paths if you use the **SHIFT_PATH_WORKFLOWS** env variable.
+
+### Plugs <--> Knobs/Inputs
+
+This is the correlation of Shift Plugs to Nuke knobs or inputs.

```

Comment:
Please add an introduction describing the fact that the Nuke plugin will read the workflow plugs from Input and Output operators. Stress on the fact that functionalities of the plugin rely on the naming of certain plugs chosen at creation time. Renaming might cause issues with the detection.

Moreover the `*` formatting is not clear. Please state in a sentence the formatting meaning for plug detection

Reply:
Note added.

Reply:
Added a comment here https://github.com/Inbibo/shift_docs/pull/93/files#r1882324057

Please review it